http.go

```diff
@@ -180,6 +151,8 @@ func probeHTTP(target string, w http.ResponseWriter, module Module, registry *pr
 	}
 
 	request, err := http.NewRequest(config.Method, target, nil)
+	request.Header.Add("HOST", request.Host)
+	request.Host = ip.String()
```

Comment:
You can just set this to the URL's original host, it'll make the header do the right thing.

You then need to change the target URL to have the IP